examples/sdp_examples/Converting a non-dcp compliant problem to Semi-Definite Programming Problem.ipynb

```diff
@@ -0,0 +1,111 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Non-DCP Compliant Problem to Semi-Definite Programming Problem \n",
+    "\n",
+    "Consider the problem\n",
+    "\n",
+    "minimize $\\frac{(c^T x)^2}{d^Tx}$\n",
+    "\n",
+    "subject to Ax +b$\\geq$ 0\n",
+    "\n",
+    "Also $d^Tx$>0 whenever Ax+b$\\geq$ 0\n",
+    "\n",
+    "**It is very easy to obseve that the objective function is neither concave or convex or affine and hence it is not dcp compliant problem.**\n",
+    "\n",
+    "** exp = square(c'*x)/d'*x **    \n",
+    "\n",
+    ">WARNING: Expression not DCP compliant. Trying to solve non-DCP compliant problems can lead to unexpected behavior.\n",
+    "\n",
+    ">WARNING: Expression not DCP compliant. Trying to solve non-DCP compliant problems can lead to unexpected behavior.\n",
+    "\n",
+    ">WARNING: Expression not DCP compliant. Trying to solve non-DCP compliant problems can lead to unexpected behavior.\n",
+    "\n",
+    ">AbstractExpr with\n",
+    "\n",
+    ">head: *\n",
+    "\n",
+    ">size: (1, 1)\n",
+    "\n",
+    ">sign: Convex.NoSign()\n",
+    "\n",
+    ">vexity: Convex.NotDcp()\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Equivalent SDP\n",
+    "\n",
+    "We introduce a new variable $t \\in \\mathbf{R}$\n",
+    "\n",
+    "Now define $D$ as follows:\n",
+    "\n",
+    "$D=\\left[\\begin{array}{cc}t&c^Tx\\\\c^Tx&d^Tx\\end{array}\\right]$\n",
+    "\n",
+    "Using the theory of Schur Complements we can show that $D \\succeq 0$ (i.e is D is positive semi-definite matrix)\n",
```

Comment:
...only for some t's! That's the most important part of the argument...! Spend a few more sentences explaining this.